app/boards/shields/leeloo_micro/README.md

```diff
@@ -0,0 +1,115 @@
+- Support for per-switch RGB underglow.
+- Support for both 110mAh or 700mAh batteries.
+- Support for Alps Alpine Micro On/off switches.
+
+# Building Leeloo-Micro ZMK Firmware
+
+ZMK Firmware: [Introduction to ZMK](https://zmk.dev/docs/)
+Installation: [Installing ZMK](https://zmk.dev/docs/user-setup)
+Customization: [Customizing ZMK](https://zmk.dev/docs/customization)
+Development Environment: [Basic Setup](https://zmk.dev/docs/development/setup)
+
+Build commands for the default keymap of Leeloo-Micro:
+
+```
+west build -d build/left -p -b nice_nano_v2 -- -DSHIELD=leeloo_micro_left
+west build -d build/right -p -b nice_nano_v2 -- -DSHIELD=leeloo_micro_right
+```
+
+Build commands for your custom keymap of Leeloo-Micro:
+
+```
+west build -d build/right -p -b nice_nano_v2 -- -DSHIELD=leeloo_micro_right -DZMK_CONFIG="C:/dev/zmk/[yourNmae]/leeloo_micro/config"
+west build -d build/left -p -b nice_nano_v2 -- -DSHIELD=leeloo_micro_left -DZMK_CONFIG="C:/dev/zmk/[yourName]/leeloo_micro/config"
+```
+
+## Building Leeloo-Micro's ZMK Firmware with nice!view Displays
+
+There are a couple of files that need to be adjusted before the build commands can be run.
+
+### Edit the leeloo_micro.keymap File
+
+Near the top 3rd of the leeloo_micro.keymap file, locate the following code block:
+
+```
+//nice_view_spi: &spi0 {
+//  compatible = "nordic,nrf-spim";
+//  pinctrl-0 = <&spi0_default>;
+//  pinctrl-1 = <&spi0_sleep>;
+//  pinctrl-names = "default", "sleep";
+//  cs-gpios = <&pro_micro 4 GPIO_ACTIVE_HIGH>;
+//};
+```
+
+Remove the forward slashes to resemble the following:
+
+```
+nice_view_spi: &spi0 {
+    compatible = "nordic,nrf-spim";
+    pinctrl-0 = <&spi0_default>;
+    pinctrl-1 = <&spi0_sleep>;
+    pinctrl-names = "default", "sleep";
+    cs-gpios = <&pro_micro 4 GPIO_ACTIVE_HIGH>;
+};
+```
+
+Save your changes and close the file.
+
+### Edit the leeloo_micro.conf file:
+
+Near the top of the leeloo_micro.conf file, locate the following line items:
+
+```
+# Uncomment the following line to enable the OLED Display or nice!view Display
+# CONFIG_ZMK_DISPLAY=y
+```
+
+Remove the # character in front of the CONFIG_ZMK_DISPLAY configuration item to resemble the following:
+
+```
+# Uncomment the following line to enable the OLED Display or nice!view Display
+CONFIG_ZMK_DISPLAY=y
+```
+
+Save your changes and close the file.
+
+### Sample Build Commands for nice!view Displays
+
+Build commands for the default keymap of Leeloo-Micro:
+
+```
+west build -d build/left -p -b nice_nano_v2 -- -DSHIELD="leeloo_micro_left nice_view_adapter nice_view"
+west build -d build/right -p -b nice_nano_v2 -- -DSHIELD="leeloo_micro_right nice_view_adapter nice_view"
+```
+
+Build commands for your custom keymap of Leeloo-Micro:
+
+```
+west build -d build/left -p -b nice_nano_v2 -- -DSHIELD="leeloo_micro_left nice_view_adapter nice_view" -DZMK_CONFIG="/workspaces/zmk-config/[yourNmae]/leeloo_micro/config"
```

Comment:
```suggestion
west build -d build/left -p -b nice_nano_v2 -- -DSHIELD="leeloo_micro_left nice_view_adapter nice_view" -DZMK_CONFIG="/workspaces/zmk-config/[yourName]/leeloo_micro/config"
```
(also on the line below)

Reply:
@caksoylar, I was able to catch a few more of these!  Thank you!